modify WEBPACKER PARALLEL environment variable. (#2850)
rails/webpacker: docs/troubleshooting.md

printf "#\041/bin/bash\n\necho \"Running elm-make with sysconfcpus -n 2\"\n\n$HOME/dependencies/sysconfcpus/bin/sysconfcpus -n 2 $HOME/your_rails_app/node_modules/.bin/elm-make-old \"\$@\"" > $HOME/your_rails_app/node_modules/.bin/elm-make
chmod +x $HOME/your_rails_app/node_modules/.bin/elm-make
```

> ## If an error occurs or times out during a production build on Continuous Integration (CI) services such as CircleCI, CodeShip, Travis CI

Reviewer: ```suggestion
## If an error occurs or times out during a production build
```

> The parallel option of Tersor-Plugin is enabled in the production build, which may cause memory shortage.

Reviewer: ```suggestion
The parallel option of TerserWebpackPlugin is enabled in the production build, which may cause memory shortage.
```

To solve this problem, you can specify the WEBPACKER_PARALLEL environment variable.
Possible values ​​are true, false, or a number.

```
RAILS_ENV=production WEBPACKER_PARALLEL=false bundle exec rake webpacker:compile
```

> Check the link for details.
> 
> https://webpack.js.org/plugins/terser-webpack-plugin/#parallel

Reviewer: ```suggestion
[`terser-webpack-plugin` documentation](https://webpack.js.org/plugins/terser-webpack-plugin/) has more details.
```
